pkg/secrets/k8s/k8s_secrets_test.go

```diff
@@ -41,6 +41,26 @@ func TestKubernetesSecrets(t *testing.T) {
 			})
 		})
 
+		Convey("Given a map of data entries and a secret with backslashes", func() {
+			m := make(map[string][]byte)
+			// need to simulate a conjur secret, with 1 backslash, so we use unicode to simulate this
```

Comment:
why do we need unicode to simulate this? can't we have `[]byte("super\secret");`?

Reply:
explaining in comments now but essentially
We need to simulate a conjur secret with 1 backslash (*\1234). When typing just  []byte("*\1234"), we get an escaping error. If we send a secret with 2 backslashes, the secret will be sent with 2 backslashes which is not presentative of the conjur secret (*\1234)
therefore a trick is to use unicode escape sequence to send the secret with just 1 backslash